.github/workflows/ci-project-linters.yml

```diff
@@ -17,6 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     env:
       GH_TOKEN: ${{ secrets.GH_TOKEN_PROJECT_ACCESS }}
+      UNSET_ISSUE: "https://github.com/HHS/simpler-grants-gov/issues/1932"
```

Comment:
Can you expand on what `UNSET_ISSUE` means here?

Reply:
Great callout! Just added some comments in this commit: [docs: Explains what UNSET_ISSUE is in ci-project-linters.yml](https://github.com/HHS/simpler-grants-gov/pull/2519/commits/7ef7e0237e089aab4b4d2233711f34ed17efd825)